cmds/cpud/main.go

```diff
@@ -390,13 +396,17 @@ func errval(err error) error {
 
 func handler(s ssh.Session) {
 	a := s.Command()
+	if *debug {
+		a = append([]string{a[0], "-d"}, a[1:]...)
+	}
 	v("handler: cmd is %v", a)
 	cmd := exec.Command(a[0], a[1:]...)
 	cmd.SysProcAttr = &syscall.SysProcAttr{Cloneflags: syscall.CLONE_NEWNS}
 	cmd.Env = append(cmd.Env, s.Environ()...)
 	ptyReq, winCh, isPty := s.Pty()
 	if isPty {
 		cmd.Env = append(cmd.Env, fmt.Sprintf("TERM=%s", ptyReq.Term))
+		cmd.Env = append(cmd.Env, "CPUD_IGNORE_CMD_ERROR=1")
```

Comment:
I gotta know what you hit that needed this but I'm good with it. 

I also realize ... damn ... the environment variables should have been CPUD_ all this time. Oh well.

Reply:
If you can, add the docs for this CPUD_ to doc.go?

Reply:
i described the issue in https://github.com/u-root/cpu/issues/40, but it turns out i had a different issue than you.  

basically bash's return value is the return value of the last command bash ran, so if you did "cpu $HOST bash", and whatever you entered happened to not return 0, (e.g. ls /no/such/file), then cpu would loudly complain about the error code.

i added a blurb to doc.go too, which we can expand with other CPUD env variables, such as the fstab stuff (?).

i also squeezed in a commit that removed cmds/**cpud**/doc.go, which was an older copy of cmds/**cpu**/doc.go